<!DOCTYPE html>
<html class="uk-notouch">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <title>EMR2023</title>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta content="width=device-width, initial-scale=1" name="viewport">
    <meta name="description">
  <?php include('cssFile.php') ?>
</head>
<body>
<?php include('header.php') ?>

<section class="uk-section">
    <div class="uk-container">
        <div class="uk-flex">
            <div class="main-div uk-width-1-1">
                <div class="textbox align-text-justify">
                    <h1>Courses</h1>

                    <p>
                        The courses will be given on the first day (May 8, 2023) of the EMR Conference. They will open
                        when ten or more participants are enrolled. Each course is limited to a maximum of fifteen participants.
                        <!--The due date to register a course is February 25, 2023.-->

                        The available courses are listed below. Click the course for details.
                    </p>
                    <ul uk-accordion>
                        <!-- Course 1 -->
                        <!--<li>
                            <a class="uk-accordion-title uk-text-bold" href="#">> Course 1: Interface between Regulation and Statistics in Drug Development</a>
                            <div class="uk-accordion-content uk-padding-small uk-padding-remove-bottom uk-padding-remove-top">
                                <div>
                                    <p style="margin-bottom: 0!important; padding-bottom: 0!important;"><bold>Instructor(s)</bold>: </p>
                                    <ul class="uk-list-bullet" style="margin-bottom: 15px!important;">
                                        <li><a href="" uk-toggle="target: #modal-birol-emir">Birol Emir</a></li>
                                        <li><a href="" uk-toggle="target: #modal-demissie-alemayehu">Demissie Alemayehu</a></li>
                                    </ul>

                                    <div id="modal-birol-emir" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p><bold>Birol Emir, PhD,</bold> is Executive Director and Statistics Lead of Real-World Evidence (RWE) at Pfizer Inc.
                                                In addition, Dr. Emir has served as Adjunct Professor of Statistics and Lecturer at Columbia University in New York
                                                and as an External PhD Committee Member at Graduate School of Arts and Sciences, Rutgers, the State University of New Jersey.
                                                Recently, his primary focuses have been on Big Data, predictive modeling, and genomic data analysis. He has numerous
                                                publications in refereed journals, and he has co-edited Statistical Topics in Health Economics and Outcome Research,
                                                published by Chapman & Hall/CRC Press. He has taught many short courses and has given several invited presentations.</p>
                                        </div>
                                    </div>

                                    <div id="modal-demissie-alemayehu" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full", style="margin-bottom: 10px;">
                                            <p> <bold>Demissie Alemayehu, PhD,</bold> is Vice President and Head of the Statistical Research and Data Science Center at Pfizer
                                                Inc. He is a Fellow of the American Statistical Association, has published widely, and has served on the editorial boards
                                                of major journals, including the Journal of the American Statistical Association and the Journal of Nonparametric
                                                Statistics. Additionally, he has been on the faculties of both Columbia University and Western Michigan University.
                                                He has co-authored a monograph entitled Patient-Reported Outcomes: Measurement, Implementation and Interpretation
                                                and co- edited another, Statistical Topics in Health Economics and Outcome Research, both published by Chapman & Hall/CRC Press.
                                            </p>
                                        </div>
                                    </div>
                                </div>

                                <p>
                                    This course is aimed at researchers in the pharmaceutical industry and in academia with interest in drug development and, hence,
                                    in the design, conduct, analysis, and reporting of clinical trials or observational studies intended for regulatory purposes.
                                    The primary focus of the course is on the intersection of statistics and regulatory affairs in the context of drug development.
                                    While the course is intended to be of primary interest to medical researchers and regulatory personnel, it may also serve as a
                                    useful reference for graduate students in the health sciences who wish to get an understanding of the statistical and regulatory
                                    issues that commonly arise in the course of drug development. The material is mostly written at a level that is accessible to
                                    readers with an intermediate knowledge of statistics.
                                </p>
                                <p>
                                    The course consists of four stand-alone sections, with each section dedicated to a specific topic in regulatory affairs and statistics.
                                    In each case, the authors have made a conscious effort to provide a survey of the relevant literature and to highlight emerging and
                                    current trends and guidelines for best practices, when the latter were available.
                                </p>
                                <p>
                                    Section 1 highlights basic statistical and regulatory issues that must be addressed in the design, analysis, and reporting of clinical trials.
                                    In particular, due attention is paid to the role of regulations and guidance  documents, and the anticipated regulatory and statistical
                                    interactions throughout the drug development continuum. Reference is also made to the evolving role of the statistician, having increasing
                                    impact and visibility, in relation to the changing regulatory and healthcare landscapes. It is also underscored that the prevailing
                                    emphasis on Big Data, modern analytics, and precision medicine will continue to present interesting challenges and opportunities both
                                    to regulatory scientists and statisticians.
                                </p>
                                <p>
                                    The primary purpose of Section 2 is to provide a thorough discussion of major statistical issues that commonly arise in the course of
                                    drug development and regulatory interactions. The section on multiplicity outlines measures that should be taken to ensure the validity
                                    of inferential results that are intended to be the basis for regulatory decision-making. In a separate section, a thorough review of
                                    best practices is provided to handle missing values, which are ubiquitous in clinical trials, with special reference to pertinent
                                    guidelines and the emergent topic of estimands. While superiority trials are common to support drug approval, there are situations
                                    where it is essential to conduct non- inferiority studies. The  regulatory requirements and underlying principles of such trials are
                                    summarized, and suggestions are provided relating to the salient points to be considered for both efficacy and safety assessment.
                                    In light of the increasing focus to accelerate drug development with speed and efficiency, a summary of a few commonly used novel
                                    approaches is provided, including adaptive and flexible designs, enrichment studies, and studies conducted under the so- called master
                                    protocols. Other topics of regulatory and statistical import covered in this section include Bayesian approaches, issues with subgroup
                                    analysis, and the assessment of benefits and risks of pharmaceutical products.
                                </p>
                                <p>
                                    Section 3 highlights the role of the statistician in the course of drug development, with especial emphasis on the skills required to
                                    ensure effective interactions with regulatory and other external bodies. In addition to the traditional tasks they perform in the design,
                                    analysis, and reporting of trials, statisticians now have an important seat in strategic decision-making, including advisory committee
                                    meetings assembled by regulatory bodies, evaluation of interim data by safety monitoring groups, and portfolio prioritization discussions
                                    by pharmaceutical executives.
                                </p>
                                <p>
                                    Finally, the focus of Section 4 is mainly on trending topics in drug development, with emphasis on the current regulatory thinking and
                                    the  associated challenges and opportunities. Although the experiences of pharmaceutical companies and regulatory bodies with some of
                                    the topics may be limited, there is a growing interest in embedding them in the current drug development paradigm. Notable examples
                                    include the role of patient-reported outcomes and their use in fostering patient-centric drug development, and the implication of the
                                    digital revolution toward advancing personalized medicine.
                                </p>

                            </div>
                        </li>-->

                        <!-- Course 2 -->
                        <li>
                            <a class="uk-accordion-title uk-text-bold" href="#">> Course 1: Introduction to explainable machine learning with examples in healthcare</a>
                            <div class="uk-accordion-content uk-padding-small uk-padding-remove-bottom uk-padding-remove-top">
                                <div>
                                    <p style="margin-bottom: 0!important; padding-bottom: 0!important;"><bold>Instructor(s)</bold>: </p>
                                    <ul class="uk-list-bullet" style="margin-bottom: 15px!important;">
                                        <li><a href="" uk-toggle="target: #modal-biecek">Przemyslaw Biecek</a></li>
                                    </ul>

                                    <div id="modal-biecek" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Przemyslaw Biecek</bold>'s goal is to support humans' effectiveness through safe, ethical, effective,
                                                and automated predictions. He implements this by developing processes, methods, tools, and software
                                                for responsible machine learning.
                                            </p>
                                            <p style="padding-bottom: 12px">
                                                He defended his PhD in mathematical statistics in 2007 and gained the full professor title in computer science in 2023.
                                                During this time he worked in various positions. As academic researcher at the Warsaw University of Technology and the
                                                University of Warsaw. As AI expert at the OECD and <a href="https://gpai.ai/">GPAI.AI</a>. As ML specialist at Samsung,
                                                IBM, Netezza and Disney. As entrepreneur, he founded the <a href="https://www.mi2.ai/">MI2.AI</a> RedTeam which offers
                                                services on training and auditing predictive models from the perspective of transparency, robustness and fairness.
                                            </p>
                                            <p style="padding-bottom: 12px">
                                                He has always been fascinated by data visualisation. He now uses this interest to work on the visualisation of predictive
                                                models. This is the topic of his latest book "Explanatory Model Analysis" <a href="https://ema.drwhy.ai/">https://ema.drwhy.ai/</a>.
                                            </p>
                                            <p style="padding-bottom: 12px">
                                                In free time, he writes stories and comics in the Beta and Bit series introducing Data Literacy to high school students
                                                <a href="https://www.mi2.ai/beta-bit.html">https://www.mi2.ai/beta-bit.html</a>.
                                            </p>
                                        </div>
                                    </div>
                                </div>

                                <p>
                                    The aim of the workshop is to introduce participants to explainable artificial intelligence (XAI) methods that can be used to build
                                    predictive models and extract knowledge from predictive models. The workshop will combine discussion of the theoretical basis together
                                    with examples with code for your own execution. We will use real-world data for a   mortality prediction problem for covid or
                                    classification problem for heart disease.
                                </p>
                                <p>
                                    The discussed methods are available in many programming languages and various libraries, but the workshop will be based on examples
                                    in R using the DALEX library. The scope of the workshop coincides with that of the book Explanatory Model Analysis
                                    <a href="https://ema.drwhy.ai/">https://ema.drwhy.ai/</a>.
                                </p>
                                <p>
                                    The first part of the workshop is dedicated to exploratory data analysis tools and preparing for modelling. The second part of the
                                    workshop is focused on tools for developing predictive models. For the purposes of the example, we will discuss decision trees,
                                    random forests and techniques for automatic tuning of random forests. The third part will focus on local model explanation techniques.
                                    We will discuss SHAP (Shapley values), break-down and LIME, the most popular methods for local exploration of models. The fourth part
                                    will be devoted to global model explanation techniques. We will discuss the permutation importance technique for variables and the
                                    Partial Dependence technique. The workshop will be based on material from <a href="https://github.com/BetaAndBit/RML">https://github.com/BetaAndBit/RML</a>
                                </p>

                                <h3>Why?</h3>
                                <p>
                                    Complex machine learning models are frequently used in predictive modeling. There are a lot of examples for random forest-like or
                                    boosting-like models in medicine, finance, agriculture, etc. But who trusts in black boxes? In this workshop we will show why and
                                    how one would analyse the structure of the black-box model. This will be a hands-on workshop. In each part there will be a short
                                    lecture and then time for practice and discussion. Using the example of analysing a specific dataset, we will show the basics of
                                    modelling with tree models. We will then show how to evaluate and analyse such models using XAI techniques. From the packages, we
                                    will learn about randomForest, party, mlr3, DALEX, modelStudio and arenar.
                                </p>
                            </div>
                        </li>

                        <!-- Course 3 -->
                        <li>
                            <a class="uk-accordion-title uk-text-bold" href="#">> Course 2: Removing unwanted variation from large-scale RNA sequencing data with PRPS</a>
                            <div class="uk-accordion-content uk-padding-small uk-padding-remove-bottom uk-padding-remove-top">
                                <div>
                                    <p style="margin-bottom: 0!important; padding-bottom: 0!important;"><bold>Instructor(s)</bold>: </p>
                                    <ul class="uk-list-bullet" style="margin-bottom: 15px!important;">
                                        <li><a href="" uk-toggle="target: #modal-molania">Ramyar Molania</a></li>
                                        <li><a href="" uk-toggle="target: #modal-trussart">Marie Trussart</a></li>
                                    </ul>

                                    <div id="modal-molania" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Ramyar Molania</bold> is a computational cancer biologist at Walter and Eliza Hall Institute (WEHI) of
                                                medical research in Australia . He received a PhD in bioinformatics from The University of Melbourne in 2018 and
                                                then he joined Professor Tony Papenfuss laboratory as a research fellow. His research interests mainly involve
                                                in developing statistical and computational tools for removal of unwanted variation from large and complicated
                                                single cell and bulk gene expression data to achieve reliable and accurate results. He is also working on integration
                                                of multimodal single cell data to better understand the treatment responses of cancer patients. He is PCF young
                                                investigator, and he works with Associate Professor Shahneen Sandhu on integrative analysis of multimodal single
                                                cell data of metastatic castration resistant prostate cancer in response to new treatments.
                                            </p>
                                        </div>
                                    </div>

                                    <div id="modal-trussart" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Marie Trussart</bold> is a senior post-doctoral researcher from the Walter and Eliza Hall Institute in
                                                Australia in the department of Bioinformatics. She got her PhD degree on System Biology and Structural
                                                Biology Modeling from the EMBL Centre for Genomic Regulation in Barcelona, Spain working on
                                                Hi-C data and 3D modelling of the chromosome structure. She joined Terry Speed’s lab as a postdoc
                                                fellow about 6 years ago and is focusing on the development of new statistical methods in large
                                                bulk RNA-seq dataset and single-cell protein data in cancer research.
                                            </p>
                                        </div>
                                    </div>
                                </div>

                                <p>
                                    Large scale datasets generated by different omics technologies present unique challenges in terms of normalization and integration.
                                    This course focuses on expanding biostatistical and bioinformatics methods for such challenges. We will be focusing on the RUV
                                    normalization methods, which have shown great promise in dealing with the challenges presented by large scale datasets from TCGA.
                                    RUV-PRPS which is a novel strategy (Molania et al, 2023, Nat. Biotech,
                                    <a href="https://www.nature.com/articles/s41587-022-01440-w#code-availability">https://www.nature.com/articles/s41587-022-01440-w#code-availability</a>)
                                    uses pseudo-replicates of pseudo-samples (PRPS) to normalize RNA-seq data in situations when technical replicate is not available.
                                    In this course we will be presenting the new RUV-PRPS package we have been developing, which is a user-friendly R package that enable
                                    researchers to run RUV-PRPS method and to visualize diagnostic plots before and after normalization to assess the quality and consistency
                                    of their data.
                                </p>

                                <p style="margin-bottom: 0!important; padding-bottom: 0!important;">
                                    <bold>Session 1:  Introduction to large-scale RNA sequencing and RUV methods - Theorical session </bold>
                                    <ul class="uk-list-bullet">
                                        <li>Introduction on Removing Unwanted Variation (RUV) methods and model</li>
                                        <li>
                                            Pseudo-replicates and pseudo-samples approach (Ramyar et al, Nature Biotech, 2023,
                                            <a href="https://www.nature.com/articles/s41587-022-01440-w#code-availability">https://www.nature.com/articles/s41587-022-01440-w#code-availability</a>)
                                        </li>
                                    </ul>
                                </p>

                                <p style="margin-bottom: 0!important; padding-bottom: 0!important;">
                                    <bold>Session 2: Identification of unwanted variation in RNA-seq data - Hands on session</bold>
                                    <ul class="uk-list-bullet">
                                        <li>RNA-seq from the Cancer Genome Atlas (TCGA) and their provided normalisations</li>
                                        <li>RUV-PRPS package with statistical methods to identify unwanted variation:
                                            <ul class="uk-list-circle">
                                                <li>Functions to identify variation in categorical variables: PCA, silhouette coefficient, ARI, ANOVA, vector correlation.</li>
                                                <li>Functions to identify variation in continuous variables: Linear regression, correlation.</li>
                                            </ul>
                                        </li>
                                    </ul>
                                </p>

                                <p style="margin-bottom: 0!important; padding-bottom: 0!important;">
                                    <bold>Session 3: How to apply RUV-PRPS - Hands on session</bold>
                                    <ul class="uk-list-bullet">
                                        <li>Selection of negative control genes</li>
                                        <li>Function from RUV-PRPS package to create pseudo-replicates of pseudo-sample (PRPS) to correct for library size, batch effects and tumour purity.</li>
                                        <li>Function from RUV-PRPS package to run RUV-PRPS method.</li>
                                    </ul>
                                </p>

                                <p style="margin-bottom: 0!important; padding-bottom: 0!important;">
                                    <bold>Session 4:  Normalisation performance assessment - Hands on session</bold>
                                    <ul class="uk-list-bullet">
                                        <li>RUV-PRPS package with statistical methods to assess the performance of normalization method:
                                            <ul class="uk-list-circle">
                                                <li>Functions to identify variation in categorical variables: PCA, silhouette coefficient, ARI, ANOVA, vector correlation.</li>
                                                <li>Functions to identify variation in continuous variables: Linear regression, correlation.</li>
                                            </ul>
                                        </li>
                                        <li>How unwanted variation can influence down-stream analysis including gene-gene correlation, survival analysis</li>
                                    </ul>
                                </p>

                            </div>
                        </li>

                        <!-- Course 4 -->
                        <li>
                            <a class="uk-accordion-title uk-text-bold" href="#">> Course 3: How to use cloud technologies for reliable and responsible Data Science projects?</a>
                            <div class="uk-accordion-content uk-padding-small uk-padding-remove-bottom uk-padding-remove-top">
                                <div>
                                    <p style="margin-bottom: 0!important; padding-bottom: 0!important;"><bold>Instructor(s)</bold>: </p>
                                    <ul class="uk-list-bullet" style="margin-bottom: 15px!important;">
                                        <li><a href="" uk-toggle="target: #modal-cosgun">Erdal Coşgun</a></li>
                                        <li><a href="" uk-toggle="target: #modal-vincent">Vincent Carey</a></li>
                                        <li><a href="" uk-toggle="target: #modal-deniz">Deniz İlhan Topcu</a></li>
                                    </ul>

                                    <div id="modal-cosgun" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Erdal Coşgun</bold> received his B.Sc from the Department of Statistics, Faculty of Science at
                                                Hacettepe University in 2007. He started his Ph.D. research as a research assistant at the Department
                                                of Biostatistics, Faculty of Medicine at Hacettepe University the same year, and completed my Ph.D.
                                                thesis titled “New Approach to Unsupervised Based Classification on Microarray Data” in 2013.
                                                He worked in the Section on Statistical Genetics, Department of Biostatistics at University of Alabama
                                                at Birmingham (UAB) for 6 months in 2009. After that, he got a 3-months bioinformatics training at the
                                                Research and Development Campus of the Pfizer Inc. in Groton, MA, by receiving a full scholarship of
                                                the company in 2011.  After finished his Ph.D., he worked as an Asst. Prof. at Acıbadem University,
                                                Faculty of Medicine, Department of Biostatistics and Medical Informatics. (Head of Department,2014-2015)

                                                He joined Microsoft in 2015 as a Global Black Belt, Technology Solutions Professional on Advanced Analytics.
                                                He was responsible for Azure Machine Learning, R Server and Stream Analytics across the MEA region. He has
                                                been working at Microsoft Genomics since December 2016 as a Senior Data and Applied Scientist.
                                                <a href="https://www.microsoft.com/en-us/research/people/ercosgun/">... more</a>
                                            </p>
                                        </div>
                                    </div>

                                    <div id="modal-vincent" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Vincent Carey</bold> is Associate Professor of Medicine (Biostatistics) in the Channing Division of
                                                Network Medicine, Brigham and Women’s Hospital, Harvard Medical School. As a Fulbright Specialist
                                                and as an invited lecturer, he has given short courses in statistical genomics on four continents.
                                                He was an inaugural faculty member in the Cold Spring Harbor Laboratory Summer Course on statistical
                                                analysis of genome-scale data, and is former Editor-in-Chief of The R Journal. He is Scientific
                                                Director of Bioinformatics in the National Institute of Allergy and Infectious Diseases Immune
                                                Tolerance Network, and is a member of the Scientific Advisory Board of the Vaccine and Immunology
                                                Statistical Center of the Collaboration for AIDS Vaccine Discovery. Vince is a co-founder of the
                                                Bioconductor project. <a href="https://pll.harvard.edu/instructor/vincent-carey">... more</a>
                                            </p>
                                        </div>
                                    </div>

                                    <div id="modal-deniz" uk-modal>
                                        <div class="uk-modal-dialog uk-modal-body uk-text-light">
                                            <button class="uk-modal-close-default" type="button" uk-close></button>
                                            <h2 class="uk-modal-title uk-text-normal">About</h2>
                                            <hr class="uk-divider-full" style="margin-bottom: 10px;">
                                            <p style="padding-bottom: 12px">
                                                <bold>Dr. Deniz İlhan Topcu</bold> is a medical doctor and biochemistry specialist with a BSc
                                                in computer engineering. With extensive training and experience in both medicine and computer science,
                                                Topcu has developed expertise in using R for clinical laboratory-related analysis. As an experienced R user,
                                                Topcu is particularly interested in the application of artificial intelligence, machine learning,
                                                and data analytics in a clinical laboratory setting. Topcu is passionate about leveraging these
                                                tools to improve patient outcomes and optimize healthcare delivery.
                                            </p>
                                        </div>
                                    </div>
                                </div>

                                <p style="padding-bottom: 5px!important;">
                                    Researchers are using cloud environments for biomedical data sharing, run analysis tools, and collaborate.
                                    In this hands-on course, we will cover the following topics:
                                </p>
                                <ul class="uk-list-decimal uk-margin-remove-top">
                                    <li>Create a reliable and secure cloud environment. Data Sharing and Auto scale of your compute solutions – <bold>60 mins</bold></li>
                                    <li>Deploy and use Jupyter Lab, VS Code Server on terra.bio – <bold>30 mins</bold></li>
                                    <li>Selected topics in genomic visualization and analysis with Bioconductor on cloud - <bold>75 mins</bold></li>
                                    <li>Responsible Data Science use-cases – <bold>15 mins</bold></bold></li>
                                </ul>

                                <h4 class="uk-margin-medium-top">Requirements:</h4>
                                <ul class="uk-list-bullet uk-margin-remove-top">
                                    <li>Mid-level (200) Linux OS experience</li>
                                    <li>Mid-Level (200) R programming experience</li>
                                    <li>Mid-Level (300) Python programming experience</li>
                                    <li>Experienced in ‘Jupyter Notebook/Lab OR Hub’ usage with different kernel types (R, Python, Julia, Spark etc.)</li>
                                    <li>Virtual Machines will be provided in the course, but participants SHOULD BRING THEIR LAPTOPS OR PC.</li>
                                </ul>

                                <br>
                                <bold>Quota</bold>: Max. 10 participants.
                                <br>
                                <bold>Time</bold>: 3 hours.

                            </div>
                        </li>
                    </ul>

                </div>

            </div>
        </div>
    </div>
</section>

<?php include('footer.php') ?>
<?php include('jsFile.php') ?>
</body>
</html>
